.github/workflows/build.yml

```diff
@@ -9,10 +9,10 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04  # TODO: This will be deprecated soon, should probably move to ubuntu-24.04
     strategy:
       matrix:
-        python-version: ["3.6", "3.7"]
+        python-version: ["3.6", "3.7"]  # TODO: Both of these have reached end of life, should probably move to 3.12+
```

Comment:
Just a couple notes about these versions being quite old.  Probably want to bump them up, and I don't see any reason that doing so would cause any issues on GitHub Actions.